ansible/library/dut_basic_facts.py

```diff
@@ -56,6 +56,8 @@ def main():
                     results['release'] = '201911'
                 elif 'master' in results['build_version']:
                     results['release'] = 'master'
+                elif 'internal' in results['build_version']:
+                    results['release'] = 'internal'
```

Comment:
What about versions like "internal-202205"?

Reply:
As we discussed offline, will use build_version to check if it's internal in condition mark file.